### PR TITLE
initial fastlane structures

### PR DESCRIPTION
**Pull request by IzzySoft**
Let's get this rolling: see #54 (F-Droid strongly prefers Fastlane in the app's repo so the app's authors can maintain it without requiring MRs on every change; my repo does also appreciate it :wink:) It might be a good idea to also move the (Android) screenshots into this structure. See my [Fastlane Cheat Sheet](https://gitlab.com/snippets/1895688) for guidance.

### CODE REVIEW

File: fastlane/android/metadata/de/short_description.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Notitzapp mit optionalem Ende-zu-Ende verschlüsseltem Sync zu Deinem PC
```
Comment (martinstoeckli) [inline fastlane/android/metadata/de/short_description.txt:1]: Are there any length restrictions? My suggestion:
Notizenapp mit optionaler Ende-zu-Ende verschlüsselter Synchronisation zu Deinem PC

**Comment by martinstoeckli** [comment]
Thank you very much for your initiative, I appreciate every support on this, because I'm not familiar with the fastlane structure.

File: fastlane/android/metadata/de/short_description.txt (hunk 1 above)
Comment (IzzySoft) [reply fastlane/android/metadata/de/short_description.txt:1]: Yes, there are – and your suggestion would be truncated to end with "Deinem". It's 80 chars. That's why I put it this way – your version was exactly what I initially started with :laughing:

What would fit, if you insist:
Notizen mit optionaler Ende-zu-Ende verschlüsselter Synchronisation zu Deinem PC
That would just scratch exactly at the borders.

**Comment by martinstoeckli** [comment]
Shall I open a branch for all fastlane changes? I can also give you access to the repository if you like.

**Comment by martinstoeckli** [comment]
@IzzySoft - I created a new branch https://github.com/martinstoeckli/SilentNotes/tree/feature/fastlane for fastlane related changes. If you want you can make changes there.

**Comment by IzzySoft** [comment]
I'm afraid this "starter kit" is all I can give in this context. I lack the time to restructure in your repo (screenshots etc), I'm already working 10h a day getting the backlog at F-Droid down. With this merged now, I can see to get your RFP processed – once you've tagged the next release that is. So I'd welcome a ping for that.

**Comment by IzzySoft** [comment]
Ah yuck, I made a mistake: it must be `fastlane/metadata/android`…

**Comment by martinstoeckli** [comment]
Thank you, I completely understand this and will have a closer look at this fastlane structure and as soon as there are updates, I will inform you. Currently I need some time for the Dropbox API change, then I can have a look at what the fastlane structure requires. Thanks again for the nudge.

**Comment by IzzySoft** [comment]
Gladly! As for now it's just summary & description, I don't think many changes will be needed. And it's simple text files, so I don't expect any issues. The formatting I've used for the full description is something I call "markdown lite", proved the best compromise between my repo (formats excellent), F-Droid (formats well) and Play (strips `<a href` if used – but I'm not sure whether it keeps the URLs or the text :see_no_evil:). Details on that [here in my wiki](https://gitlab.com/IzzyOnDroid/repo/-/wikis/Fastlane) – basically: keep each paragraph on a single line, separate paragraphs by a single empty line, and separate bullet-point lists from surrounding blocks by an empty line. Apply simple HTML formatting where appropriate :wink:

**Pull request merged**